scripts/phar_build.sh

```diff
@@ -5,11 +5,24 @@ set -e
 echo "Installing composer dependencies with --no-dev..."
 composer install --no-dev
 
-echo "Building terminus.phar..."
+VERSION=$(cat .version)
+## if not in the context of a github action, lable the build as
+## a dev build
+if [[ -z "${GITHUB_RUN_ID}" ]]
+then
+    echo "$VERSION+dev" > .version
+fi
+
+BUILD_VERSION=$(cat .version)
+
+echo "Building terminus.phar...${BUILD_VERSION}"
```

Comment:
@greg-1-anderson Building locally will add a "+dev" to the version ID. Building in GitHub actions produces a version without it.

<img width="516" alt="Screenshot 2023-04-17 at 4 53 02 PM" src="https://user-images.githubusercontent.com/119924/232634977-2569d55a-65d9-4b02-97fe-b2faec4165ed.png">

Reply:
<img width="350" alt="Screenshot 2023-04-17 at 4 55 08 PM" src="https://user-images.githubusercontent.com/119924/232635058-66218452-5a1d-4b9a-a669-e3a37570aca4.png">

Reply:
It should be `-dev`; per semver.org, `-` indicates the "pre-release" portion. `+` indicates the "build number" portion, which is simply a monotonically increasing number unrelated to the other portions of the version string.